.github/workflows/build-test-distribute.yml

```
@@ -241,6 +245,61 @@ jobs:
          path: MeshLib.nupkg
          retention-days: 1
 
+     - name: Configure AWS Credentials
+       uses: aws-actions/configure-aws-credentials@v4
+       with:
+         role-to-assume: arn:aws:iam::259351611210:role/GitHubMeshLibAwsCredsRole
+         aws-region: us-east-1
+
+     - name: Upload NuGet Package to S3
+       shell: powershell
+       run: aws s3 cp MeshLib.nupkg 's3://test-artifacts-git/MeshLib.nupkg' --quiet
+
+  start-unity-instance:
```

can we move this job to `unity-nuget-test.yaml`?

---

another reason to integrate these jobs into a single file is to simplify start/stop of the unity test instance
```suggestion
  start-unity-instance:
    if: ${{ github.event_name == 'schedule' }}
```